src/frontend/src/app/asset-readings/assets/assets.component.css

```diff
@@ -0,0 +1,9 @@
+table.scroll {
```

Comment:
I'm more concerned about responsiveness; why a tablet and 27" monitor should have same height for scrollable container; we should make it conditional per viewport